netdata/netdata
# Fake collector to provoke ACLK messages

##### Summary

A fake collector that can be used to provoke the messages needed in the integration testing.

##### Component Name
ACLK

##### Test Plan

1. Install.
2. `aclk/tests/install-fake-charts.d.sh`  (may need to `chmod +x`)
3. Run the agent
4. Check the messages at the VerneMQ using the paho inspection and/or your own testing.

##### Additional Information
